android/app/build.gradle

```diff
@@ -97,18 +97,14 @@ def enableSeparateBuildPerCPUArchitecture = true
 def enableProguardInReleaseBuilds = false
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "26.0.1"
+    compileSdkVersion 26
```

Comment:
Not sure why not 27?  It's out, and we have it in our CI build image.

Reply:
Ah, because this is what Android Studio fixed it to for me.

Reply:
Alright.  Well, we can ship like this for now.

Reply:
I'm about to push and update to 27

Reply:
Nevermind, not going to 27 tonight:

> GC overhead limit exceeded